static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js

```diff
@@ -25,12 +25,25 @@ define([
     var GustyleComcontent = function ($adSlot, params) {
         this.$adSlot = $adSlot;
         this.params  = params;
+        this.contentPosition = {
+            bottom: 'gu-display__content-position--bottom',
+            top: 'gu-display__content-position--top'
+        };
+        this.fontSize = {
+            small: 'gu-display__content-size--small',
+            regular: 'gu-display__content-size--regular',
+            big: 'gu-display__content-size--big'
+        };
     };
 
     GustyleComcontent.prototype.create = function () {
         var externalLinkIcon = svgs('externalLink', ['gu-external-icon']),
             templateOptions = {
-                clickMacro: this.params.clickMacro,
+                articleContentColor: this.params.articleContentColor === 'white' ? 'gu-display__content-color--bright ' : 'gu-display__content-color--dark',
```

Comment:
If you change this in DFP to bright and dark then you can do the same as I've mentioned below